solr/contrib/ltr/src/java/org/apache/solr/ltr/ModelQuery.java

```diff
@@ -202,7 +209,7 @@ public ModelWeight createWeight(IndexSearcher searcher, boolean needsScores, flo
     final FeatureWeight[] modelFeaturesWeights = new FeatureWeight[modelFeatSize];
     List<FeatureWeight > featureWeights = new ArrayList<>(features.size());
     
-    if(LTRThreadModule.getMaxThreads() <= 1 || LTRThreadModule.getMaxQueryThreads() <= 1){
+    if(ltrThreadMgr == null || ltrThreadMgr.getMaxThreads() <= 1 || ltrThreadMgr.getMaxQueryThreads() <= 1){
```

Comment:
I wouldn't support a null threadManager.  Can you just construct a default one that has 1 thread set so you dont have to check nulls?